cdc-sink-test/src/main/java/com/hazelcast/jet/tests/cdc/sink/CdcSinkTest.java

```diff
@@ -0,0 +1,238 @@
+        int sleep = sleepMsBetweenItem;
+        int preserveItem = preserveItemMod;
+
+        Pipeline pipeline = Pipeline.create();
+
+        StreamSource<ChangeRecord> source = SourceBuilder
+                .stream("srcForCdcSink", procCtx -> new int[1])
+                .<ChangeRecord>fillBufferFn((ctx, buf) -> {
+                    int messageId = ctx[0];
+                    int entryId = messageId / 3;
+                    int messageType = messageId % 3;
+                    ChangeRecord changeRecord;
+                    if (messageType == 0) {
+                        // insert
+                        changeRecord = new ChangeRecordImpl(0, messageId, "{\"id\":" + entryId + "}",
+                                "{\"id\":" + entryId + ",\"name\":\"name" + entryId + "\","
+                                + "\"__op\":\"c\",\"__ts_ms\":1588927306264,\"__deleted\":\"false\"}");
+                    } else {
+                        if (messageType == 1) {
+                            // update
+                            changeRecord = new ChangeRecordImpl(0, messageId, "{\"id\":" + entryId + "}",
+                                    "{\"id\":" + entryId + ",\"name\":\"name" + entryId + "_" + entryId + "\","
+                                    + "\"__op\":\"u\",\"__ts_ms\":1588927306264,\"__deleted\":\"false\"}");
+                        } else {
+                            if (entryId % preserveItem == 0) {
+                                // For every "preserveItemMod" item do not remove it from the map.
+                                // This should happen ~ once per minute when default params are used.
+                                changeRecord = new ChangeRecordImpl(0, messageId, "{\"id\":" + entryId + "}",
+                                        "{\"id\":" + entryId + ","
+                                        + "\"name\":\"name" + entryId + "_" + entryId + "_" + entryId + "\","
+                                        + "\"__op\":\"u\",\"__ts_ms\":1588927306264,\"__deleted\":\"false\"}");
+                            } else {
+                                // delete
+                                changeRecord = new ChangeRecordImpl(0, messageId, "{\"id\":" + entryId + "}",
+                                        "{\"id\":" + entryId + ",\"name\":\"name" + entryId + "_" + entryId + "\","
+                                        + "\"__op\":\"d\",\"__ts_ms\":1588927306264,\"__deleted\":\"true\"}");
+                            }
+                        }
+                    }
+                    buf.add(changeRecord);
+                    ctx[0]++;
+                    sleepMillis(sleep);
+                })
+                .createSnapshotFn(ctx -> ctx[0])
+                .restoreSnapshotFn((ctx, state) -> ctx[0] = state.get(0))
+                .build();
+
+        Sink<ChangeRecord> sink;
+        if (clusterName.contains(DYNAMIC_CLUSTER)) {
+            sink = CdcSinks.remoteMap(SINK_MAP_NAME + clusterName,
```

Comment:
so we always use the map in stable cluster for cdc sink, then why are we running the test on both cluster ?